products/jhipster.md

```diff
@@ -0,0 +1,72 @@
+---
+title: JHipster
+category: app
+permalink: /jhipster
+iconSlug: NA
+versionCommand: yo jhipster
```

Comment:
According to https://www.baeldung.com/jhipster this is the command to create a new app, so it is not suitable for the `versionCommand`. Is there a `jhipster -v` or `jhipster --version` command ?